docs/units-and-global-variables.rst

```diff
@@ -262,6 +262,15 @@ For more information, see the section on :ref:`address`.
     to make safe Ether transfers, always check the return value of ``send``, use ``transfer`` or even better:
     Use a pattern where the recipient withdraws the money.
 
+.. warning::
+    Due to the fact that the EVM considers a call to a non-existing contract to always succeed,
+    Solidity includes an extra check using the ``extcodesize`` opcode when performing external calls.
```

Comment:
I think this is mentioned somewhere else in the docs, maybe you could like that part?

Reply:
It is, but the issue was still open despite this, so I thought you wanted it to be repeated here too? See https://github.com/ethereum/solidity/pull/11176#discussion_r601765598

Reply:
Sorry, I meant to say "link", just in case the other description is longer, but we also don't have to link.

Reply:
Well, it's shorter but I think a link is a good idea anyway. I added one.